dub.sdl

```diff
@@ -21,12 +21,3 @@ configuration "library-manual-version" {
 configuration "library-applink" {
 	targetType "sourceLibrary"
 }
-
-configuration "unittest" {
-	targetType "executable"
-	targetName "openssl-test-library"
-	dflags "-main"
-	excludedSourceFiles "source/deimos/openssl/applink.d"
-	preGenerateCommands `${DUB} scripts/generate_version.d` platform="posix"
-	versions `DeimosOpenSSLAutoDetect`
-}
```

Comment:
Have you checked what happened for a project depending on `openssl` ? Specifically, if it auto-detects the version or uses the fallback ?

Reply:
`dub test` in `examples/sslecho` seems to work just as fine as on master, does that count?

Reply:
I guess it does :)